README.md

```diff
@@ -35,10 +35,24 @@ Note: This works on Windows, Linux, and MacOS, provided the prerequisites are in
 cmake -S . -B build -DUBPF_ENABLE_TESTS=true
 cmake --build build --config Debug
 ```
-
 ## Running the tests
-### Linux and MacOS
+
+### Linux and MacOS native
+```
+cmake --build build --target test --
+```
+
+### Linux aarch64 cross-compile
+Note: This requires qemu and the aarch64 toolchain.
 ```
+# Build bpf_conformance natively as a workaround to missing boost libraries on aarch64.
+cmake -G Ninja -S external/bpf_conformance -B build_bpf_conformance
+cmake --build build
+# Build ubpf for aarch64
+cmake -G Ninja -S . -B build -DUBPF_ENABLE_TESTS=true -DUBPF_SKIP_EXTERNAL=true \
```

Comment:
I had to set the [`CMAKE_TRY_COMPILE_TARGET_TYPE`](https://cmake.org/cmake/help/latest/variable/CMAKE_TRY_COMPILE_TARGET_TYPE.html) variable to `STATIC` -- I am sure that was just related to my setup, but I wanted to flag it in case it helps!